From: Ichthyostega <prg@ichthyostega.de>
Date: Thu, 17 May 2018 17:37:49 +0200
Subject: Fix Linux Version information

...looks like this was just overlooked...

also disable the BETA_RELEASE flag, since this package is based on a release version
---
 Common/clientversion.h | 6 +++---
 configure.ac           | 2 +-
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Common/clientversion.h b/Common/clientversion.h
index 0df1e26..88b866d 100644
--- a/Common/clientversion.h
+++ b/Common/clientversion.h
@@ -17,8 +17,8 @@
 		#define	CLIENT_VERSION_PRETTY		"macOS 3.0.2"
 		#define CLIENT_SETTINGS				"ElectricSheep"
 	#else
-		#define CLIENT_VERSION				"LNX_2.7b33"
-		#define	CLIENT_VERSION_PRETTY		"version 2.7b33"
+		#define CLIENT_VERSION				"LNX_3.0.2"
+		#define	CLIENT_VERSION_PRETTY		"version 3.0.2"
 		#define CLIENT_SETTINGS				"ElectricSheep"
 	#endif
 #endif
@@ -33,7 +33,7 @@
 #define REDIRECT_SERVER				"community.sheepserver.net"
 #define REDIRECT_SERVER_FULL		"https://community.sheepserver.net"
 
-#define	BETA_RELEASE
+//#define	BETA_RELEASE
 
 
 #endif // CLIENTVERSION_H_INCLUDED
diff --git a/configure.ac b/configure.ac
index e9a562c..eb158d9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,7 +4,7 @@ AC_PREREQ(2.57)
 ifdef([m4_pattern_allow],
       [m4_pattern_allow([^AC_.*])])
 
-AC_INIT(electricsheep,2.7b33-svn,http://code.google.com/p/electricsheep/issues/list)
+AC_INIT(electricsheep,3.0.2,http://code.google.com/p/electricsheep/issues/list)
 AC_CANONICAL_HOST
 AC_CANONICAL_TARGET
 
